llvm/test/CodeGen/AMDGPU/GlobalISel/llvm-intrinsics/reduce/add.ll

```diff
@@ -0,0 +1,2388 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Don't need all the -verify-machineinstrs in these tests.

Also, do we even have test coverage for these for the DAG? Should we just merge the dag/gisel tests into one in the main test directory?

Reply:
removed